draft_release_notes.md

```diff
@@ -0,0 +1,18 @@
+```markdown
+## <Feature> added to DT-UI
+**Description of Previous Approach**
```

Comment:
Is this a standard when using a component library Previous Approach, New Approach, Benefits?

I thought release notes followed a pattern like this:
`# Upgrade Steps
- [ACTION REQUIRED]
-

# Breaking Changes
- 
- 

# New Features
-
-

# Bug Fixes
-
-

# Improvements
-
-

# Other Changes
-
-`

Reply:
This looks better to me...